app/models/call_list.rb

```diff
@@ -0,0 +1,20 @@
+# Object representing a patient's call list.
+class CallList
```

Comment:
Suggestion: `CallList` implies a has_many patients relationship to me. Maybe `CallListEntry` instead?